tests/Concerns/ManagesDataDefinitionsTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Copyright 2019 Colopl Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Colopl\Spanner\Tests\Concerns;
+
+use Colopl\Spanner\Connection;
+use Colopl\Spanner\Tests\TestCase;
+use Illuminate\Database\Events\QueryExecuted;
+use Illuminate\Support\Facades\Event;
+
+class ManagesDataDefinitionsTest extends TestCase
+{
+    public function test_createDatabase_with_statements(): void
+    {
+        $events = Event::fake([QueryExecuted::class]);
+
+        $conn = new Connection('test-instance', 'test_' . time(), '', ['instance' => 'test-instance']);
+
+        if (!empty(getenv('SPANNER_EMULATOR_HOST'))) {
+            $this->setUpEmulatorInstance($conn);
+        }
```

Comment:
`setUpDatabaseOnce()` cannot be used?

```suggestion
        $this->setUpDatabaseOnce($conn);
```

Reply:
setUpDatabaseOnce will create a database, which is not what we want.

We want to create the instance but not the database.